JKGalaxies/glua/framework/minigames/pazaak/pazaak.lua

```diff
@@ -1075,10 +1084,19 @@ function PazaakGame:CleanUp()
 	timer.Remove(self.TimeoutName)
 	timer.Remove(self.TimerName)
 	
+	--remove player invincibility: futuza
+	self.Players[1].Player.GodMode = false
+	self.Players[1].Player.NoKnockback = false
+	self.Players[2].Player.GodMode = false
+	self.Players[2].Player.NoKnockback = false
+	print( "Invulnerability disabled for " .. tostring(self.Players[1].Player:GetName()) .. " and " .. tostring(self.Players[2].Player:GetName()) )
```

Comment:
I left it in because it's still part of a "test" function (pzktest), and only shows up on the server side.  Also because I may end up changing the solution to just quit pazaak if damaged, rather then this whole invulnerability bit.